workspaces/adventure-pack/goodies/typescript/Number.prototype.chr/index.test.ts

```diff
@@ -0,0 +1,54 @@
+import { describe, expect, it } from "@jest/globals";
+
+import "./index";
+
+describe("Number.prototype.chr", () => {
+  it("convert ASCII codepoint to ASCII string", () => {
+    expect(Number(65).chr()).toBe("A");
```

Comment:
Btw, most of the values in this test are already numbers, so passing them through the Number function doesn't do anything. So for example, the following code would run successfully:

```ts
const value = 65;
expect(value.chr()).toBe("A");
```

The dream, perhaps, would be to be able to write simply `65.chr()` directly. This works in languages like Ruby. In JavaScript the main reason it doesn't work is because the parser doesn't quite know how to deal with this, but wrapping the value in parentheses does the trick, for example:

```ts
expect((65).chr()).toBe("A");
```

Take a look at the test for [`Number.prototype.digits`](https://github.com/code-chronicles-code/leetcode-curriculum/blob/main/workspaces/adventure-pack/goodies/typescript/Number.prototype.digits/index.test.ts) for inspiration, where this was also necessary.